src/components/Jobs/components/Job/Job.js

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedDate, FormattedTime, FormattedMessage } from 'react-intl';
+import classNames from 'classnames';
+
+import { Button } from '@folio/stripes/components';
+
+import Progress from '../Progress';
+import jobMetaTypes from './jobMetaTypes';
+
+import css from './Job.css';
+
+class Job extends React.Component {
+  static defaultProps = {
+    handlePreview: () => {
+      // TODO: to be implemented in further stories
+    },
+  };
+
+  static propTypes = {
+    job: PropTypes.shape({
+      jobExecutionHRID: PropTypes.string.isRequired,
+      jobProfileName: PropTypes.string.isRequired,
+      fileName: PropTypes.string.isRequired,
+      startedDate: PropTypes.string.isRequired,
+      completedDate: PropTypes.string.isRequired,
+      status: PropTypes.string.isRequired,
+      runBy: PropTypes.shape({
+        firstName: PropTypes.string.isRequired,
+        lastName: PropTypes.string.isRequired,
+      }).isRequired,
+      progress: PropTypes.shape({
+        current: PropTypes.number.isRequired,
+        total: PropTypes.number.isRequired,
+      }).isRequired,
+    }).isRequired,
+    handlePreview: PropTypes.func,
+    checkDateIsToday: PropTypes.func.isRequired,
+  };
+
+  formatTime(dateStr) {
+    const isToday = this.props.checkDateIsToday(dateStr);
+    const datePart = !isToday && <FormattedDate value={dateStr} />;
+    const timePart = <FormattedTime value={dateStr} />;
+    const todayPart = isToday && <FormattedMessage id="ui-data-import.today" />;
+
+    return (
+      <span>{datePart} {timePart} {todayPart}</span>
+    );
+  }
+
+  getRootClasses() {
+    return classNames(css.delimiter, css.jobHeader);
+  }
+
+  render() {
+    const { job } = this.props;
+    const {
+      jobProfileName,
+      fileName,
+      status,
+      jobExecutionHRID,
+      runBy: { firstName, lastName },
+      progress: { current, total },
+    } = job;
+    const jobMeta = jobMetaTypes[status](job);
+    const dateLabel = `ui-data-import.${jobMeta.dateLabel}Running`;
```

Comment:
As I understand it is not a label. It is a key or translation id. Maybe we should rename it?

Reply:
Do you mean to remove variable and put it like that? If so, I agree.

Reply:
I am ok with variable, but it is not a label, it is label id or key. I suggest to rename it for example to dateLabelId?

Reply:
Agree, will rename.